src/cheapestTree.ts

```diff
@@ -1,16 +1,30 @@
 import { calculateTreeQuantity } from './calculateTreeQuantity'
 import { calculateTreePrices } from './calculateTreePrices'
 import { calculateTreeCraftFlags } from './calculateTreeCraftFlags'
-import { RecipeTree, RecipeTreeWithCraftFlags } from './types'
+import { RecipeTree, RecipeTreeWithCraftFlags, RecipeTreeWithPrices } from './types'
 import { NestedRecipe } from '@gw2efficiency/recipe-nesting'
 
 export function cheapestTree(
   amount: number,
   tree: NestedRecipe,
   itemPrices: Record<string, number>,
   availableItems: Record<string, number> = {},
-  forceBuyItems: Array<number> = []
+  forceBuyItems: Array<number> = [],
+  valueOwnItems: string = 'false'
```

Comment:
I think this makes sense to be `false` (not the string). Let's just use the user's material settings, `BUY | SELL` is overkill for this (we can add it later if requested)